dev/io.openliberty.microprofile.health.4.0.internal/src/io/openliberty/microprofile/health40/internal/AppTracker40Impl.java

```diff
@@ -82,4 +85,40 @@ public void applicationStarted(ApplicationInfo appInfo) throws StateChangeExcept
         }
     }
 
+    /**
+     * Returns the MBeanInfo of appName if the ApplicationMBean exists, otherwise null.
+     *
+     * @return the MBeanInfo of appName if the ApplicationMBean exists, otherwise null.
+     */
+    @Override
+    protected String getApplicationMBean(String appName) {
```

Comment:
where is this being called?

Reply:
It's called by either the the `applicationStopped(ApplicationInfo)` or `isInstalled(String)`  from the parent `AppTrackerImpl` . The `isInstalled()` is called during health checks.